migrations/postgres/20240328152300_task_events_table.sql

```diff
@@ -11,4 +11,4 @@ CREATE TABLE task_events (
 );
 
 CREATE INDEX task_events_task_id_idx ON task_events (task_id);
-CREATE INDEX task_events_dataset_id_idx On task_events(dataset_id);
\ No newline at end of file
+CREATE INDEX task_events_dataset_id_idx On task_events(dataset_id);
```

Comment:
Q: just to double-check, in `task_events` table, we have nullable `dataset_id` field, is that expected?

Reply:
They are only stored in "created" events.